assessment.py

```diff
@@ -177,10 +215,16 @@ def GET(self, bank_id):
     def PUT(self, bank_id):
         try:
             am = autils.get_assessment_manager()
+            data = self.data()
```

Comment:
For my own information: Does this change indicate a problem in the previous syntax, or is this just so that you can more easily debug what's going on?